.github/workflows/claim-namespace.yml

```
@@ -44,7 +46,7 @@ jobs:
         name: Namespace API request
         uses: JamesIves/fetch-api-data-action@v2
         with:
-          endpoint: https://open-vsx.org/api/${{steps.get_namespace.outputs.namespace}}
+          endpoint: https://open-vsx.org/api/${o{steps.get_namespace.outputs.namespace}}
```

```suggestion
          endpoint: https://open-vsx.org/api/${{steps.get_namespace.outputs.namespace}}
```

my daughter typed this... sry